### ISSUE TITLE
An important issue regarding the way .bashrc.bak is created

### ISSUE THREAD

**Issue by shaief** [Bash-it/bash-it#453]
While running the installation script for the first time, it ends with three questions:

```
Would you like to enable all, some, or no aliases? Some of these may make bash slower to start up (especially completion). (all/some/none) all
Would you like to enable all, some, or no plugins? Some of these may make bash slower to start up (especially completion). (all/some/none) all
Would you like to enable all, some, or no completion? Some of these may make bash slower to start up (especially completion). (all/some/none) all
```

After the last one, the script just ends. When running it again due to misunderstanding of the result, the script run over the `.bashrc.bak` and deletes the original `.bashrc.bak`.

I thing it's a bad practice and destructive, and could be avoided by:
1. Using a user notification like: `"Script ended successfully! enjoy!"`
2. Ask the user for permission before running over the `.bashrc.bak`.

**Comment by shaief** [comment]
I provided a pull request that solves it:
https://github.com/Bash-it/bash-it/pull/454

**Comment by nwinkler** [comment]
Fixed with #458
